### ISSUE TITLE
Why heatmap size is also downsample into 4 times smaller

### ISSUE THREAD

**Issue by KeepGoing2019HaHa** [leoxiaobin/deep-high-resolution-net.pytorch#31]
Hi, may I ask why heatmap size is also 4x smaller than input image? I mean if you are keeping high resolution feature maps all the time, why do not just generate heatmaps with original input size for training and inference?

**Comment by YinRui1991** [comment]
I think it's probably empirical. I find some other codes also use 4x smaller heatmap. I think it's OK to use other ratio, but the model should be trained  with new output size.

**Comment by KeepGoing2019HaHa** [comment]
@YinRui1991 yes, almost all pose estimators use 4x downsample. But this paper aims to maintain high resolution feature maps, why not also generate high resolution heatmaps?

**Comment by sunke123** [comment]
@KeepGoing2019HaHa 
In this paper, we focus on maintaining the high-resolution feature maps, not increasing the resolution.
So, we maintain the high-resolution feature maps as used in most works, 4x feature maps.
You can try to use higher resolution feature maps, such as 2x or full-resolution. I think that is also helpful.
Thanks for your attention.

**Comment by HuAndrew** [comment]
I guess their work is different from other network, like hourglass, simple baseline.
In 2018 ECCV 2018 simple baseline papers,  feature maps is smaller than 4x feature maps. Using 4x feature maps fusion is well to balance parameters and performance.

**Comment by aliYaHf** [comment]
@sunke123 hi ,Can you provide training logs on the MPII dataset? I don't know why the AR value of my training is very low.

**Comment by luhengjie** [comment]
@sunke123 Hello,is the groundtruth heatmap's resolution also 4 times smaller than input image's resolution? If what I think is true,how can I modify the groundtruth heatmap's resolution? Looking forward to your reply.

**Comment by kuldeepbrd1** [comment]
> @sunke123 Hello,is the groundtruth heatmap's resolution also 4 times smaller than input image's resolution? 
The ground truth heatmaps are of the same size you input in the yaml file. However, the predicted heatmap is 1/4th of original image in height and width.

You'd have to make changes to the way layers are configured in `models.pose_hrnet`, probably the transition 3 between stage 4 and the final layer.
